docs/latest/design/roadmap.md

```diff
@@ -34,11 +34,17 @@ contributing to the project.
 
 ### [v0.3.0][v0.3.0]: Drive Advanced Features through Extension Mechanisms
 
-- Global Rate Limiting
-- AuthN/AuthZ- [Issue #336][336].
-- Lets Encrypt Integration
+- Support extended Gateway API fields.
```

Comment:
Can we link this to a specific issue like other goals in v0.3.0? This will be more clear for a roadmap.

Reply:
It should be linked to https://github.com/envoyproxy/gateway/pull/529 and  https://github.com/envoyproxy/gateway/issues/670. IMO https://github.com/envoyproxy/gateway/issues/671 is a stretch for v0.3, so feel free to add it to v0.4 if you feel confident in delivering the feature.

Reply:
I think @Xunzhuo was referring to create a specific epic, created this one https://github.com/envoyproxy/gateway/issues/707 and will link it